main.go

```diff
@@ -61,6 +69,35 @@ func RootCmd() *cobra.Command {
 	return rootCmd
 }
 
+func generateConfigs(tagsArg string, kinesisStreamNameArg string) FluentbitConfig {
+	tags, err := getTags(jsonConfigPath, tagsArg)
+	check(err, "tags not found")
+
+	kinesisStreamName, err := getKinesisStreamName(tags, kinesisStreamNameArg)
+	check(err, "kinesis stream name not found")
+
+	systemdUnit, systemdUnitLookupError := getSystemdUnit(tags)
+
+	if systemdUnitLookupError != nil {
+		fmt.Printf("%s: %v\n", "Application log shipping will not be configured", systemdUnitLookupError)
+		placeholders := map[string]string{"KINESIS_STREAM": kinesisStreamName, "APPLICATION_LOGS": ""}
+		config := replaceReplaceholders(fluentbitConfig, placeholders, tags)
+		return FluentbitConfig{
+			MainConfigFile: config,
+			ApplicationConfigFile: "",
```

Comment:
This works fine in practice (it adds an empty config file to the instance, but nothing references it so it doesn't really matter), but it feels wrong! In Scala I would obviously use an `Option`/`None` here. 

That said, I noticed that when you want to return an error in `go`, you actually have to do something like:

```go
return "", fmt.Errorf("Kinesis Stream name was not found: no LogKinesisStreamName tag, and the --kinesisStreamName arg was empty.")
```
So perhaps the aversion to returning an empty string that isn't needed wouldn't be so strong for `go` developers - what do you think @nicl?!